packages/app/src/app/pages/Dashboard/Components/Folder/FolderCard.tsx

```diff
@@ -38,7 +38,7 @@ export const FolderCard: React.FC<FolderItemComponentProps> = ({
       width: '100%',
       height: 240,
       backgroundColor: 'grays.700',
-      border: '1px solid',
+      border: 'transparent',
```

Comment:
We can remove this border property, below this line we also set the `borderColor` separately which can be removed as well.